acs-integration-tests/pom.xml

```diff
@@ -1060,24 +1042,7 @@
                                 <ZAC_UAA_URL>https://${uaa.cf.domain}.${cf.base.domain}</ZAC_UAA_URL>
                                 <ACS_UAA_URL>https://${uaa.cf.domain}.${cf.base.domain}</ACS_UAA_URL>
                                 <ZAC_URL>https://${zac.cf.domain}.${cf.base.domain}</ZAC_URL>
-                                <!-- ACS zones (a.k.a tenancies) used for 
-                                    the tests -->
                                 <ZONE1_UAA_URL>https://${uaa.zone}${uaa.cf.domain}.${cf.base.domain}</ZONE1_UAA_URL>
-                                <ZONE2_UAA_URL>not-used</ZONE2_UAA_URL>
-                                <!-- OAuth clients and users used for the 
-                                    tests -->
-                                <ACS_CLIENT_ID>${ACS_CLIENT_ID}</ACS_CLIENT_ID>
-                                <ACS_CLIENT_SECRET>${ACS_CLIENT_SECRET}</ACS_CLIENT_SECRET>
-                                <ACS_TEST_USER>${ACS_TEST_USER}</ACS_TEST_USER>
-                                <ACS_TEST_USER_PWD>${ACS_TEST_USER_PWD}</ACS_TEST_USER_PWD>
-                                <APM_CLIENT_ID>not-used</APM_CLIENT_ID>
-                                <APM_CLIENT_SECRET>not-used</APM_CLIENT_SECRET>
```

Comment:
Is the acceptance goal executed on the pipeline?

Reply:
@sebastiantb, acceptance tests are executed by the deployment pipeline agains deployed application.